### PR TITLE
Add Blue Portals Pack

### PR DESCRIPTION
**Pull request by TheDarkBomber**
* Add Soundtrack from BP
* Add Cube Catcher

Checklist on TheDarkBomber/BEE2-items-TheDarkBomber-Contributions#12

### CODE REVIEW

File: packages/bpItems/cube_catcher/items/p1/editoritems.txt [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+					"BrushSideCount"	"286"
+				}
+			}
+
+			"TargetName"		"prop_boxcatcher"
+			"Outputs"
+			{
+				"CONNECTION_STANDARD"
+				{
+					"Activate"		"instance:prop_boxcatcher_relayok&i;OnUser2"
```
Comment (TeamSpen210) [inline packages/bpItems/cube_catcher/items/p1/editoritems.txt]: I'd suggest you fix up the names, so it doesn't use prefab naming.

Comment (TheDarkBomber) [inline packages/bpItems/cube_catcher/items/p1/editoritems.txt]: Prefab naming?

**Comment by TeamSpen210** [comment]
You'll need to revert the readme changes, since we don't want those to affect the merge.

**Comment by TeamSpen210** [comment]
Not by deleting it, resoring the current one...

**Comment by TheDarkBomber** [comment]
There is no readme in the `dev` branch...

File: packages/bpItems/cube_catcher/items/p1/editoritems.txt (hunk 1 above)
Comment (TeamSpen210) [inline packages/bpItems/cube_catcher/items/p1/editoritems.txt]: &i

**Comment by TheDarkBomber** [comment]
This will need `Mod Resource` and `New Pack/Enhancement` labels.

**Comment by Luke18033** [comment]
So, it looks like we don't actually have permission to use the Blue Portals stuff. They said to wait until the anniversary edition is out, and use the resources from that.

**Pull request closed without merging**